docs/functions/functions.json

```diff
@@ -368,6 +368,34 @@
 				true
 			]
 		},
+		{
+			"name": "get",
+			"id": "get",
+			"unary": false,
+			"category": "other",
+			"role": "function",
+			"description": "Get a value of a group by key. For example, `get(group(bar:4), bar)` will return `4`.\nIf the first parameter is a collection, all values of the given key are returned. For example, `get(list(group(id:\"foo\", name:\"fooname\"), group(id:\"bar\", name:\"barname\")), name)` returns `list(\"fooname\", \"barname\")`.\nIf the first parameter is a collection and the second parameter is in the form `\"key=value\"`, then the group(s) that satisfies this condition will be returned as a list. For example, `get(list(group(id:\"foo\", name:\"fooname\"), group(id:\"bar\", name:\"barname\")), \"name=fooname\")` returns `list(group(id:\"foo\", name:\"fooname\"))`.\nYou can attach a **key** to the end of `get()` with a `.` operator to get the value of a key. For example, `get(list(group(id:\"foo\", name:\"fooname\"), group(id:\"bar\", name:\"barname\")), \"id=foo\").name` returns `\"fooname\"`.",
+			"argument": [
+				{
+					"name": "object",
+					"optional": false,
+					"type": "Group or List of Groups",
+					"description": "Where to look up keys and find values."
+				},
+				{
+					"name": "key",
+					"optional": false,
+					"type": "Text",
+					"description": "The key to look for or the condition to match if in the form `\"key=value\"`."
+				},
+				{
+					"name": "meta",
```

Comment:
As I mentioned in my comment, please remove this. It's for internal use.

Reply:
OK. I did see your comment though. Maybe I forgot to remove it when I was editing.